dash/include/dash/halo/StencilOperator.h

```diff
@@ -84,6 +84,34 @@ class StencilOperatorInner {
     }
   }
 
+  /**
+   * Returns the result of the given operation done on all stencil point
+   * elements and the center.
+   * The stencil points are multiplied with their coefficent (\ref StencilPoint)
+   * and the center is  multiplied with the given center coefficient.
+   * No test for halo elements. -> useful only for elements with stencil
+   * points pointing to inner elements. (no halo elements)
+   *
+   * \param coords center coordinate
+   * \param coefficient for center
+   * \param op operation to use (e.g. std::plus). default: std::plus
+   */
+  ElementT get_value_at(
+    const ElementCoords_t& coords, ElementT coefficient_center,
+    std::function<ElementT(const ElementT&, const ElementT&)> op =
```

Comment:
Make the function a template parameter, e.g., `BinaryFunc`, to avoid creating a `std::function` from lambdas